src/containers/score.js

```diff
@@ -81,7 +80,7 @@ class Score extends Component {
 				<div id={this.props.uri} className="scorepane">
 					<div className="controls" />
 					<div className="annotations" />
-					<InlineSVG className="score" src={ svg } />
+					<div dangerouslySetInnerHTML={{__html: svg}} />
```

Comment:
one difference here is that InlineSVG uses `<i>` which is an inline element while I've used `<div>` here, which is a block element. I'm not sure if it'll cause problems.

Reply:
Thanks!